README.md

```diff
@@ -48,6 +51,26 @@ To run the application in `development` mode run:
 npm run dev
 ```
 
+### Aws localstack
+
+#### Install and run LocalStack AWS
+
+- Install [LocalStack AWS CLI](https://github.com/localstack/awscli-local)
+- Run AWS LocalStack Docker container:
+
+```bash
+docker run --pull=always -d -p 4566:4566 -p 4510-4559:4510-4559 localstack/localstack:latest
+```
+
+#### Setup local buckets
+
+You need local buckets setup in localstack
+
+```bash
+awslocal s3 mb s3://cdp-uploader-quarantine --endpoint-url http://localhost:4566
+awslocal s3 mb s3://my-bucket --endpoint-url http://localhost:4566
```

Comment:
Are you not missing a `=`? after the alias. Also if using localstack/localstack the default endpoint is https, whilst with cdp-local-env it is http

Reply:
(ignore the `=` comment. Removed it)

Reply:
Though if using awslocal the endpoint-url is superfluous. As all it does is append it to the raw `aws` command

Reply:
nope - all ok - https://docs.localstack.cloud/user-guide/aws/mwaa/#create-a-s3-bucket